### ISSUE TITLE
Crashes on startup if the window is big enough

### ISSUE THREAD

**Issue by p4block** [vinszent/gnome-twitch#321]
I'm using a tiling window manager (bspwm) that will force a window to be of a specific size on its creation, ignoring hints. 

Opening gnome-twitch on my 3840x2160 screen (minus 35 pixels from bottom bar) will cause it to crash instantly with this log:

```
pablo@arch ~ % gnome-twitch
[11:39:23] Message - GNOME-Twitch : {GtApp:370} Startup, running version 'r1066.24635c4'
[11:39:23] Message - GNOME-Twitch : {GtApp:339} Activate
[11:39:23] Message - GNOME-Twitch : {GtPlayer:132} Loading chat settings
[11:39:23] Message - GNOME-Twitch : {GtPlayer:999} Loaded player backend 'MPV OpenGL player backend'
[11:39:23] Message - GNOME-Twitch : {GtPlayerBackendMpvOpenGL:401} Init

ERROR:../src/gt-twitch.c:730:gt_twitch_top_channels: assertion failed (n <= 100): (102 <= 100)
[1]    18764 abort (core dumped)  gnome-twitch
```

Opening the window in smaller container, ie by having another window already open, will avoid the crash. 
If the window is resized to max size after startup it won't crash, but changing the current view will do it:

`ERROR:../src/gt-twitch.c:854:gt_twitch_top_games: assertion failed (n <= 100): (162 <= 100)`

Same thing happens on GNOME.

So basically I have to keep in mind that I cannot maximize gnome-twitch unless I'm watching a stream, and then make the window small again if I switch categories.

**Comment by vinszent** [comment]
Right, Twitch only allows you to request max 100 streams. I put a pretty stupid assertion there, thinking this would only occur from a programming error, but didn't really think about high res screens. Will fix on dev branch and try to merge soon.
